docker-compose-sgx.yaml

```diff
@@ -25,17 +25,17 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    environment:
+        - SGX_MODE=HW
         - TCF_DEBUG_BUILD=${TCF_DEBUG_BUILD:-}
         - DISPLAY=${DISPLAY:-}
+        - MAKECLEAN=${MAKECLEAN:-1}
     volumes:
-      - ./:/project/TrustedComputeFramework
+      # Below volume mappings are required for DISPLAY settings by heart disease GUI client
       - /tmp/.X11-unix:/tmp/.X11-unix
+      - ~/.Xauthority:/root/.Xauthority
     devices:
       - "/dev/isgx:/dev/isgx"
     working_dir: "/project/TrustedComputeFramework/tools/build"
```

Comment:
working_dir may not be needed here as build is done inside Dockerfile ?

Reply:
Done